.github/workflows/add-package-label.yml

```diff
@@ -0,0 +1,56 @@
+name: Add Package Label
+on:
+  issues:
+    types: [opened, edited]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const {
+              repo: { owner, repo },
+              payload: {
+                action,
+                issue: { body, labels: currentLabels, number: issue_number },
+              },
+            } = context;
+
+            if (!body) {
+              console.log("could not determine the issue body");
+              return;
+            }
+
+            // NOTE: assumes all packages will be in the @esri NPM scope
+            const packageRegex = /(?<=\[X\]\s@esri\/)[\w-]*$/gm;
+            const packages = body.match(packageRegex);
+
+            if (packages?.length) {
```

Comment:
We could eliminate this check, and its associated nesting, by providing an empty array as a fallback when assigning `packages` above:

```ts
const packages = body.match(packageRegex) || [];
```